src/database/seeders/20201202225250-user.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable implicit-arrow-linebreak */
+module.exports = {
+	up: (queryInterface) =>
+	/*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+      Example:
+      return queryInterface.bulkInsert('People', [{
+        name: 'John Doe',
+        isBetaMember: false
+      }], {});
+      */
+		queryInterface.bulkInsert('Users', [
+			{
+				fullname: 'liliane Nishimwe',
+				country: 'Rwanda',
+				city: 'kigali',
+				gender: 'female',
+				birthdate: '2000-6-2',
+				isVerified: 'true',
+				tel: '0856438287',
+				profilePicture: 'avatar.jpg',
+				email: 'lnishimwe@daviscollege.com',
+				password: '$2y$12$RftUykWaCUKDOuXe3qx.WO5eRx7pvGeiaqyW0egkiEUWy35Nc9BEi',
+				role: 'Requester',
```

Comment:
can you use a user that are already in the seed with the same role to avoid creation so many seeds